frontend/lib/resolution/default-functions.cpp

```diff
@@ -185,19 +186,27 @@ generateInitSignature(Context* context, const CompositeType* inCompType) {
 
   // push all fields -> formals in order
   for (int i = 0; i < rf.numFields(); i++) {
-    auto qualType = rf.fieldType(i);
-    auto name = rf.fieldName(i);
-    bool hasDefault = rf.fieldHasDefaultValue(i);
-    const uast::Decl* node = nullptr;
-
-    auto fd = UntypedFnSignature::FormalDetail(name, hasDefault, node);
+    auto fieldQt = rf.fieldType(i);
+    auto formalName = rf.fieldName(i);
+    bool formalHasDefault = rf.fieldHasDefaultValue(i);
+    const uast::Decl* formalAst = nullptr;
+
+    // A field may not have a default value. If it is default-initializable
```

Comment:
Did this fix have anything to do with running the dyno scope resolver? If not, it arguably should be in a different PR.

Reply:
Yes. I need to run the full resolver, and init calls failing to resolve were screwing up unit tests.